app.js

```diff
@@ -1,10 +1,39 @@
+/*jslint node: true */  
 'use strict';
 
 var ari = require('ari-client');
 var util = require('util');
 var sla = require('./lib/sla.js');
+var Q = require('q');
 
-ari.connect('http://127.0.0.1:8088', 'user', 'pass', clientLoaded);
+var connect = Q.denodeify(ari.connect);
 
-function clientLoaded (err, client){
-    console.log(sla());}
+connect('http://127.0.0.1:8088', 'user', 'pass')
+  .then(function (client) {
+    clientLoaded(client);
+  })
+  .catch(errHandler);
+
+/**
+ * Starts Stasis app 'sla' and initiates SLA application after ARI connection.
+ * @param {Object} client - Object that contains information from the ARI 
+ *   connection.
+ */
+function clientLoaded (client) {
+  client.start('sla');
+  sla(client)
+    .then(console.log)
+    .catch(errHandler)
+    .finally(function () {
+      process.exit(0);
```

Comment:
Since this is in a finally handler, process will exit with a non error 0 code even in the case of an error.